src/components/follow.tsx

```diff
@@ -25,6 +25,28 @@ const styles = (kicker: string, inverted: string): SerializedStyles => css`
 	border: none;
 	background: none;
 	margin-left: 0;
+	margin-top: 4px;
+
+	svg {
+		width: 24px;
+		height: 24px;
+		margin-bottom: -6px;
```

Comment:
Worth converting these to `rem` for scaling reasons (the scale slider)?

Reply:
I checked with the templates (that we're trying to mimic) and they don't scale with the slider so decided to go with px. What do you think?

Reply:
We're not trying to mimic the scaling behaviour of the templates. They _only_ scale body text. We scale everything (with the exception of the headline).